AudioLink/RenderTextures/dont_use_audio_link.renderTexture

```diff
@@ -0,0 +1,37 @@
+%YAML 1.1
```

Comment:
Also here, the name implies to me that this is something that should go into AudioLinkSandbox, and not the main folder

Reply:
This was also part of the dead-ended path.